terraform/20-app/elasticache.front-end.tf

```diff
@@ -0,0 +1,25 @@
+module "front_end_elasticache_security_group" {
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "5.1.0"
+
+  name   = "${local.prefix}-front-end"
+  vpc_id = module.vpc.vpc_id
+
+  ingress_with_source_security_group_id = [
+    {
+      description              = "frontend tasks to cache"
+      rule                     = "redis-tcp"
+      source_security_group_id = module.ecs_service_front_end.security_group_id
+    }
+  ]
+}
+
+resource "aws_elasticache_serverless_cache" "front_end_elasticache" {
+  engine = "redis"
+  name   = "${local.prefix}-front-end"
```

Comment:
There is an annoying name length restriction from the elasticache service, bit this follows our naming convention anyhow